de_web_plugin.cpp

```diff
@@ -7586,6 +7598,12 @@ void DeRestPluginPrivate::updateSensorNode(const deCONZ::NodeEvent &event)
 
                             if (ia->id() == 0x0021) // battery percentage remaining
                             {
+                                // filter endpoint
+                                if (event.endpoint() != i->fingerPrint().endpoint)
+                                {
+                                    continue;
+                                }
```

Comment:
I'm not sure if this could break battery reporting for devices other than Danfoss Ally.
To be safe, I'd propose to limit this to VENDOR_DANFOSS-